### ISSUE TITLE
How to construct a console with styles?

### ISSUE THREAD

**Issue by darrenburns** [willmcgugan/rich#9]
Hi! 👋 

I've defined a console and theme like so. I think there's some out of date docs around constructing consoles, so I've read the code and this is what I got to:

```python
from rich.theme import Theme, Style as St

theme = Theme({
    "pass-tag": St.parse("white on green"),
    "fail-tag": St.parse("white on red"),
    "skip-tag": St.parse("white on blue"),
    "xfail-tag": St.parse("white on magenta"),
    "xpass-tag": St.parse("white on yellow"),
    "dryrun-tag": St.parse("white on green"),
    "test-location": St.parse("dim white"),
})

console = Console(theme=theme)
```

Is this the intended usage?

P.S. Happy to make a PR updating the docs if so! :)

**Comment by willmcgugan** [comment]
👋Docs are a work in progress.

Yes, that works, but it replaces all the default styles. If you want to merge your styles, don't set `theme` in the constructor, instead call `push_styles`.

```python
console.push_styles({
    "pass-tag": St.parse("white on green"),
    "fail-tag": St.parse("white on red"),
    "skip-tag": St.parse("white on blue"),
    "xfail-tag": St.parse("white on magenta"),
    "xpass-tag": St.parse("white on yellow"),
    "dryrun-tag": St.parse("white on green"),
    "test-location": St.parse("dim white"),
})
```

**Comment by willmcgugan** [comment]
Documentation is fixed in v0.6.0, and there is a slight change to how it works. If you construct a Theme, it will merge the default styles by default -- so you won't need to call `push_styles`.

Please close if it works for you.
